File: Solution_Practice_I/python/prob1_7.py
while True:
    print(f"Ingrese la distancia y cuanto tiempo queda → (distancia time)")
    distance, time = map(float, input().split(" "))
    if distance >= 0 and time > 0:
        break

time *= 60
print(f"{round(distance * 100 / time) / 100} m/s")

File: Solution_Practice_I/java/prob1_7.java
import java.util.Scanner;
public class prob1_7 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double distance, time;
        do {
            System.out.println("Ingrese la distancia y cuanto tiempo queda");
            distance=input.nextInt();
            time=input.nextInt();
        } while (distance<0 || time<=0);
        time*=60;
        System.out.print(Math.round(distance*100/time)/100.0 +" m/s");
        input.close();
    }
}